tests/end-to-end/cli/log-events-verbose-text-invalid-argument.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+phpunit --no-output --log-events-verbose-text logfile.txt
+--FILE--
+<?php declare(strict_types=1);
+$traceFile = 'dasdasdasdasd/dasdasdsa.log';
```

Comment:
This is not portable: please use `$traceFile = tempnam(sys_get_temp_dir(), __FILE__);` instead.

Reply:
Hi, i need an invalid path here to test fail error messages, i hope this is ok:
```$traceFile = sys_get_temp_dir() . '/invalid-directory/invalid.file';```

Reply:
My bad, sorry. Yes, please use what you just proposed.